mempool/clist_mempool.go

```diff
@@ -24,12 +24,11 @@ import (
 // mempool uses a concurrent list structure for storing transactions that can
 // be efficiently accessed by multiple concurrent readers.
 type CListMempool struct {
-	// Atomic integers
-	height   int64 // the last block Update()'d to
-	txsBytes int64 // total size of mempool, in bytes
+	height   atomic.Int64 // the last block Update()'d to
+	txsBytes atomic.Int64 // total size of mempool, in bytes
```

Comment:
The issue only mentions the data race for `height` field. Did you also discover that there are data races for `txsBytes` and `notifiedTxsAvailable` fields?

Reply:
The issue uses height just as an example. But yes, in the tests for this PR, I also saw that there were data races in the other variables.

Reply:
`txBytes` was actually implemented correctly. It was declared as an `int64` and accessed with `atomic.*` methods. I changed its type to `atomic.Int64` to keep it consistent with the other variables.